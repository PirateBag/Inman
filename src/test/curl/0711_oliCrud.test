 6    1       0  10.00   0.00  2025-0801  2025-0803 OPEN   MO INSERT
 7    2       6  10.00   0.00  2025-0801  2025-0803 OPEN   MO INSERT
 8    3       6  10.00   0.00  2025-0801  2025-0803 OPEN   MO INSERT
 9    4       6  10.00   0.00  2025-0801  2025-0803 OPEN   MO INSERT
10    6       6  10.00   0.00  2025-0801  2025-0803 OPEN   MO INSERT
11    7       6  80.00   0.00  2025-0801  2025-0803 OPEN   MO INSERT
12    8       6  40.00   0.00  2025-0801  2025-0803 OPEN   MO INSERT
13    9       6  40.00   0.00  2025-0801  2025-0803 OPEN   MO INSERT

Change order 6 from Open to Closed.  Components should get closed as well.


===JSON===

{ "rows" : [
        {  "crudAction" : "CHANGE",  "id" : 6, "itemId" : "1", "parentOliId" : "0", "quantityOrdered" : "10", "quantityAssigned" : "0.0",
        "startDate" : "2025-0801", "completeDate" : "2025-0803",  "orderState" : "CLOSED",
        "orderType" : "MOHEAD" }
] }