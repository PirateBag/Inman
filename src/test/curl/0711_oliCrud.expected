{
  "responseType" : "MULTILINE",
  "errors" : [ {
    "key" : 1,
    "message" : "Updated 7 line items to state CLOSED"
  } ],
  "data" : [ {
    "id" : 6,
    "crudAction" : "CHANGE",
    "itemId" : 1,
    "quantityOrdered" : 10.0,
    "quantityAssigned" : 0.0,
    "startDate" : "2025-0801",
    "completeDate" : "2025-0803",
    "parentOliId" : 0,
    "orderState" : "CLOSED",
    "orderType" : "MOHEAD"
  } ]
}